app/helpers/users_helper.rb

```diff
@@ -1,2 +1,16 @@
 module UsersHelper
+    def logged_in?
+        User.find_by(remember_token: cookies[:remember_token])
+    end
+    
+    def user
+        User.find_by_remember_token(cookies[:remember_token])
```

Comment:
[A .find or .find_by_ is never called in a view or view helper.](http://matthewpaulmoore.com/post/5190436725/ruby-on-rails-code-quality-checklist)

Reply:
This action is checking whether the user is logged in or not. The article you referred to is saying that "there’s a 99% chance there’s a better way to do it - and you should do it." 
I don't see how else to check logged in user, do you recommend any better way to do it?